.github/workflows/perf_sysbench.yml

```diff
@@ -44,6 +44,12 @@ jobs:
       - name: Add tarantool path to $PATH
         run: echo "${PWD}/src:${PWD}/extra/dist" >> ${GITHUB_PATH}
 
+      - name: Get tarantool version
+        id: tarantool-version
+        run: >
+          echo "::set-output name=value::$(tarantool -v | grep -e 'Tarantool' |
+          grep -oP '\s\K\S*')"
```

Comment:
Nit: The following commands doing the same, but look simpler:

```shell
$ tarantool -v | head -n 1 | awk '{ print $2; }'
$ tarantool -v | head -n 1 | cut -d' ' -f2
$ tarantool -e 'print(_TARANTOOL)'
# if you're worrying about old versions w/o gh-5040 fix
$ tarantool -e 'print(_TARANTOOL) os.exit()'
```

Reply:
I chose the shortest (`tarantool -e 'print(_TARANTOOL)'`). Thanks!